src/common/mixins/TrackerLayerMixin.js

```diff
@@ -250,6 +250,9 @@ const TrackerLayerMixin = (Base) =>
      * @param {number} zoom
      */
     startUpdateTime(zoom) {
+      if (this.updateTimeInterval) {
+        clearInterval(this.updateTimeInterval);
+      }
```

Comment:
You can call this.stopUpdateTime() instead

Reply:
Good idea!